keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentArtifactVeto.kt

```diff
@@ -0,0 +1,29 @@
+package com.netflix.spinnaker.keel.core.api
+
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+import java.time.Instant
+
```

Comment:
I'm confused about the 3 classes here -- can you please add some descriptions?

Reply:
I can for the first one, i'll remove the middle one (i didn't need it but forgot about it), and the third is just a lift from another class to not be in with the pin stuff